backend/src/routes/middleware/expressLogger.ts

```diff
@@ -9,15 +9,16 @@ import log from '../../services/log.js'
 const morganLog = promisify(
   morgan<any, any>(
     (tokens, req, res) => {
+      const info = {
+        url: tokens.url(req, res),
+        method: tokens.method(req, res),
+        'response-time': tokens['response-time'](req, res),
+        status: tokens.status(req, res),
+        'content-length': tokens.res(req, res, 'content-length'),
+      }
       req.log.trace(
-        {
-          url: tokens.url(req, res),
-          method: tokens.method(req, res),
-          'response-time': tokens['response-time'](req, res),
-          status: tokens.status(req, res),
-          code: 'approval',
-        },
-        tokens.dev(morgan, req, res),
+        info,
+        `${info.method} ${info.url} ${info.status} ${info['response-time']} ms - ${info['content-length']}`,
```

Comment:
I don't think you want a space between response time and 'ms' here.

Reply:
I just blindly copied the morgan format but we're not restricted by that so I'll update to remove the space.